webui/react/src/pages/F_ExpList/F_ExperimentList.settings.ts

```diff
@@ -1,13 +1,16 @@
 import * as t from 'io-ts';
+import { literal, union } from 'io-ts';
```

Comment:
nit: either reference the parts you need as `t.literal`, etc, or import them specifically, not both

Reply:
I didn't really get this one...